sdk/eventhub/azure-eventhubs/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -15,21 +15,21 @@ def create_async_eventhub_producer_client():
     event_hub_connection_str = os.environ['EVENT_HUB_CONN_STR']
     event_hub = os.environ['EVENT_HUB_NAME']
     producer = EventHubProducerClient.from_connection_string(conn_str=event_hub_connection_str,
-                                                             event_hub_path=event_hub)
+                                                             eventhub_name=event_hub)
     # [END create_eventhub_producer_client_from_conn_str_async]
 
     # [START create_eventhub_producer_client_async]
     import os
     from azure.eventhub import EventHubSharedKeyCredential
     from azure.eventhub.aio import EventHubProducerClient
 
-    hostname = os.environ['EVENT_HUB_HOSTNAME']
+    fully_qualified_namespace = os.environ['EVENT_HUB_HOSTNAME']
     event_hub = os.environ['EVENT_HUB_NAME']
```

Comment:
could rename this also to `event_hub_name` to match the pattern

Reply:
Fixed sample!